### ISSUE TITLE
[client styling] Apply Conversation Pane styling

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-client#272]
**Epic:** UX Parity between prototypes & coded client (#261) 

**This Story:** Conversation Pane Styles

**Reference materials:** tbd, but likely https://zpl.io/V4OMBzy

---
**Details, This Story:**
- Note: Nina needs to post colors here for Journalist ID Badges... will do once Branding/Color Schema is complete (via [SD-UX 36](https://github.com/freedomofpress/securedrop-ux/issues/36)).

**Comment by eloquence** [comment]
Let's treat the file-related actions as out of scope for the first iteration.

**Comment by eloquence** [comment]
A lot of progress has been made towards this; @ninavizz let's make sure we have follow-up issues for remaining polish tasks, and then we can close this issue.

**Comment by eloquence** [comment]
This is very out of date and lacks obvious actions, so closing in favor of clearly scoped follow-up issues.
